lib/runcible/models/ostree_importer.rb

```diff
@@ -4,7 +4,7 @@ class OstreeImporter < Importer
       ID = 'ostree_web_importer'.freeze
       REPO_TYPE = 'OSTREE'.freeze
 
-      attr_accessor 'branches'
+      attr_accessor 'branches', 'depth'
```

Comment:
should depth have a default value?  or is nil okay and what does nil signify? everything?